generator/app.go

```diff
@@ -84,12 +84,24 @@ type GithubTeams struct {
 	Description string
 }
 
+// Repo represents a specific repo or subdirectories owned by the group
+type Repo struct {
+	Name        string
+	Description string
+	URL         string
+	Owners      []string
+}
+
 // Subproject represents a specific subproject owned by the group
 type Subproject struct {
 	Name        string
 	Description string
+	Label       string
+	Chairs      []Person
+	Repos       []Repo
```

Comment:
I don't think this is bad, but I can't think of an example of a subproject that has multiple repos.

Reply:
API Machinery has a bunch: https://github.com/kubernetes/community/tree/master/sig-api-machinery#subprojects